### ISSUE TITLE
issue with using TeX input-method

### ISSUE THREAD

**Issue by soonhokong** [alpaker/fill-column-indicator#53]
I understand that it's **not** recommended to use `fci-mode` with non-ascii characters. But I'm wondering whether there can be a quick-fix for the problem that I have.
## Symptom

I try to input 'λ' using TeX input-method. When I type "\lambda", it shows the intermediate text after the FCI. 

```
                                                                                 |\lamb
```

Once it's completed, emacs puts 'λ' back into the point where cursor is at.

```
λ                                                                                |
```
## How to reproduce
- enable `fci-mode`
- `set-input-method` `RET` `TeX`
- Type "\lambda"

Interestingly, if I type something in the middle of a line, there is no problem. This problem only happens when I type at the end of a line.

**Comment by alpaker** [comment]
Just FYI, I'm abandoning this package and welcoming requests to be take over as maintainer.

**Comment by alpaker** [comment]
The functionality provided by this package has now been implemented natively in Emacs as `display-fill-column-indicator-mode`, available as of version 27.0.90. Unless you are forced to use an older Emacs, you should use the native implementation instead of `fci-mode`. The former is unaffected by most of the compatibility issues that this add-on has.
